mempool/src/tests/fuzzing.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    core_mempool::{CoreMempool, TimelineState},
+    shared_mempool::{peer_manager::PeerManager, tasks, types::SharedMempool},
+};
+use libra_config::config::NodeConfig;
+use libra_types::transaction::SignedTransaction;
+use proptest::{
+    prelude::*,
+    strategy::{Just, Strategy},
+};
+use std::{
+    collections::HashMap,
+    sync::{Arc, Mutex, RwLock},
+};
+use storage_interface::mock::MockDbReader;
+use vm_validator::mocks::mock_vm_validator::MockVMValidator;
+
+/// Proptest strategy for fuzzing input to mempool incoming transactions endpoint
+pub fn mempool_incoming_transactions_strategy(
+) -> impl Strategy<Value = (Vec<SignedTransaction>, TimelineState)> {
+    (
+        proptest::collection::vec(proptest::arbitrary::any::<SignedTransaction>(), 100),
```

Comment:
actually I think you should give a range as second argument to `vec` here so that it creates a vector of arbitrary size in that range, I think `0..100` would work? I haven't thought much about 100 and if it's too high :D maybe @sunshowers has an opinion on this